src/testing.js

```diff
@@ -9,3 +9,10 @@ function fibonacci(num) {
     return fibonacci(num - 1) + fibonacci(num - 2);
   }
 }
+
+var x = 5;
+try {
+  console.log("Fibonacci number #" + x + " is " + fibonacci(x));
+} catch (error) {
+  console.log("Encountered error: " + error);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this logging statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:4950/coding_rules#rule_key=javascript%3AS2228)